example/simple_testing/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
```

Comment:
For testing this, what I recommend doing is:

 - Adding `callbacks` to `BackendAttrSupport`.
 - Setting it to true in the C backend, and only the C backend.
 - Using `feature_tests`, and using `cfg(not(supports = callbacks), disable)` on any method that uses callbacks. Make a separate `callbacks.rs` test file.